docs/servo_power_test/src/graph.ts

```diff
@@ -4,10 +4,14 @@ import {AnnotationData, PowerData} from './power_test_controller';
 
 export class Graph {
   private ui: Ui;
-  public g = new Dygraph('graph', [], {});
+  public g: Dygraph;
   public annotations: dygraphs.Annotation[] = [];
-  constructor(ui: Ui) {
+  constructor(ui: Ui, divName: string) {
```

Comment:
I think passing HtmlDivElement instead of string of the div's id is better (as Dygraph takes both of them):
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/dygraphs/index.d.ts#L1160